packages/lib/survey/service.ts

```diff
@@ -662,7 +662,9 @@ export const createSurvey = async (environmentId: string, surveyBody: TSurveyInp
       }),
     };
 
-    await subscribeOrganizationMembersToSurveyResponses(environmentId, survey.id);
+    if (createdBy) {
```

Comment:
can we move this below the cache revalidation to make sure we first finish this in case this call throws an error?